### ISSUE TITLE
CakePHP assumes or requires plugin.php but CRUD has no such file

### ISSUE THREAD

**Issue by FirstWarden** [FriendsOfCake/crud#597]
Using cakephp 3.6.5

When installing the CRUD plugin, and going to the index page of a controller, I get the error 

**The plugin class `Crud\Plugin` cannot be found. Ensure your autoloader is correct.**

cakephp uses a file named 'Plugin.php': see also https://book.cakephp.org/3.0/en/plugins.html#plugin-objects
I created a Plugin.php file in friendsofcake\crud\src to see if this would alleviate the problem and it does, meaning I can now use an url like e.g. /some-controller/index.json.

I don't know if this file is mandatory for the 3.6.5 version of cake?

plugin.php
```
<?php

namespace Crud;

use Cake\Core\BasePlugin;
use Cake\Core\PluginApplicationInterface;

class Plugin extends BasePlugin
{
    public function middleware($middleware)
    {
// Add middleware here.
        return $middleware;
    }

    public function console($commands)
    {
// Add console commands here.
        return $commands;
    }

    public function bootstrap(PluginApplicationInterface $app)
    {
// Add constants, load configuration defaults.
// By default will load `config/bootstrap.php` in the plugin.
        parent::bootstrap($app);
    }

    public function routes($routes)
    {
// Add routes.
// By default will load `config/routes.php` in the plugin.
        parent::routes($routes);
    }
}

?>
```

**Comment by saeideng** [comment]
please use  `Plugin::load()` or wait for cakephp 3.6.6

even having `Plugin` class in this plugin can solve problem

**Comment by ADmad** [comment]
This issue will be fixed in next release of cake 3.6, no change is necessary in this plugin.
